data/json/npcs/talk_tags.json

```diff
@@ -11,7 +11,9 @@
       "Thorazine?  I wouldn't if I were you.",
       "You don't need thorazine, it's limiting you.",
       "Thorazine…  That's what 'they' use to keep you tame.",
-      "Don't.  This thorazine seriously clouds your mind.  You need to stay sharp."
+      "Don't.  This thorazine seriously clouds your mind.  You need to stay sharp.",
+      "That stuff isn't going to do you any good.",
+      "Nasty stuff to get hooked on, <name_g>.  You'd best get rid of it."
```

Comment:
Hallucinations says those line when they see thorazine anywhere, if the sentence does not explicitly mention thorazine it's going to be difficult to figure out what tey're talking abou.

Reply:
Would you recommend making every line for a specific drug contain the name of the drug then to avoid any confusion?